stdlib/uuid.ksy

```diff
@@ -0,0 +1,49 @@
+meta:
+  id: uuid
+  endian: le
+  license: Unlicense
+doc: "UUID according to https://www.ietf.org/rfc/rfc4122.txt"
```

Comment:
I think I would object to adding `uuid` type automatically to every UUID one might encounter. It is always unnecessary for parsing purposes (I doubt something in any format will ever depend on contents of internal UUID fields), and, 99.9% of the time, it is completely unnecessary for practical purposes: they're intended to be used as 128-bit unique IDs, and that's it.

Knowing internal structure of UUIDs is needed mostly to debug generation of UUIDs to ensure their uniqueness, or for forensic purposes. For both of that, you need to know which family of UUIDs you're debugging. Unfortunately, contrary to what RFC 4122 states, it's not always possible to determine it automatically, i.e. by analyzing bits of 8th byte. There are quite a few implementations that use completely random 128 bits as UUIDs, thus everything can happen.

So, I'd vote to do good quality implementation of UUIDs, but (1) keep variants and layout separate, in different ksy files, application of particular format to be chosen manually, (2) not encourage to import them in formats unless necessary.